### PR TITLE
EN6-197 implement advanced searching functionality

### PR DESCRIPTION
**Pull request by ichalagashvili**
Currently blocked by BE issue EN6-207

### CODE REVIEW

**Comment by ichalagashvili** [comment]
This PR contains code for:

- Implementing advanced searching functionality
- Fixing feedback received in EN6-200 jira ticket

File: src/sass/assets/AssetsAdvancedFilter.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  &__left-column {
+    padding-right: 5px;
+    @media (max-width: 768px) {
+      padding-left: 0;
+    }
+  }
+
+  // &__right-column {
```
Comment (gidesan) [inline src/sass/assets/AssetsAdvancedFilter.scss]: please remove commented code :)

File: src/ui/categories/filter/CategoryTreeFilter.js [hunk 2]
```diff
@@ -10,18 +11,27 @@ class CategoryTreeFilter extends React.Component {
...
 
+  componentDidMount() {
+    const { onDidMount } = this.props;
+    onDidMount();
+  }
+
+
   onExpandTree() {
-    const { onExpandCategory } = this.props;
+    // const { onExpandCategory } = this.props;
```
Comment (gidesan) [inline src/ui/categories/filter/CategoryTreeFilter.js]: please remove commented code

File: src/ui/categories/filter/CategoryTreeFilter.js [hunk 3]
```diff
@@ -10,18 +11,27 @@ class CategoryTreeFilter extends React.Component {
...
     const { treeExpanded } = this.state;
     this.setState({
       treeExpanded: !treeExpanded,
     });
-    onExpandCategory('home');
+    // onExpandCategory('home');
```
Comment (gidesan) [inline src/ui/categories/filter/CategoryTreeFilter.js]: please remove commented code

**Pull request merged**